csharp/src/Glacier2/ZeroC.Glacier2.props

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright (c) ZeroC, Inc. -->
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <ItemDefinitionGroup>
+        <SliceCompile>
+            <IncludeDirectories>%(IncludeDirectories);$(MSBuildThisFileDirectory)..\slice</IncludeDirectories>
```

Comment:
Should this be a forward `/`? These props are cross-platform.

Reply:
I think we should actually fix this to normalize the path we pass to the task. We can do it from the .targets files when we call the task.

Reply:
I updated the tools package to normalize paths with `[MSBuild]::NormalizePath`